dashboard/app/cache.go

```diff
@@ -15,6 +15,13 @@ import (
 )
 
 type Cached struct {
+	Open       int
```

Comment:
You can replace these 3 lines with just `SubsystemStats`.
Can also make it possible to create some helper functions that operate on SubsystemStats.